### PR TITLE
BUGZ-746: About tab: Better CPU/GPU info reporting; New copied info (including OS version)

### PR DESCRIPTION
**Pull request by zfox23**
[BUGZ-746](https://highfidelity.atlassian.net/browse/BUGZ-746)

Also adds a quick feature to our SUI Buttons which allows you to do this:
0OdF8csM8Y

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20106/HighFidelity-Beta-PR15829-3e2e020.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20106/HighFidelity-Beta-Interface-PR15829-3e2e020.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20106/HighFidelity-Beta-PR15829-3e2e020.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20106/HighFidelity-Quest-Beta-PR15829-3e2e020.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20106/HighFidelity-Beta-PR15829-3e2e020.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20106/HighFidelity-Beta-Interface-PR15829-3e2e020.exe

File: interface/resources/qml/hifi/simplifiedUI/settingsApp/about/About.qml [hunk 1]
```diff
@@ -206,12 +228,29 @@ Flickable {
...
+        var gpuModel = gpu.model;
+        if (gpuModel.length === 0) {
+            gpuModel = "Unknown";
+        }
+
+        textToCopy += "GPU: " + gpuModel + "\n";
+        textToCopy += "VR Hand Controllers: " + (PlatformInfo.hasRiftControllers() ? "Rift" : (PlatformInfo.hasViveControllers() ? "Vive" : "None")) + "\n";
+        
+        textToCopy += "\n**All Platform Info**\n";
+        textToCopy += JSON.stringify(JSON.parse(PlatformInfo.getPlatform()), null, 4);
```
Comment (MiladNazeri) [inline interface/resources/qml/hifi/simplifiedUI/settingsApp/about/About.qml]: Lines 133, 179, 232, 243, 253 are all parse functions.  Should there be any parse checks before continuing or do you feel ok that we won't get back anything funky to stop the operation flow?

Comment (zfox23) [reply interface/resources/qml/hifi/simplifiedUI/settingsApp/about/About.qml]: I feel confident that we'll be fine, but great question.

**Pull request merged**